pop-api/examples/fungibles/tests.rs

```diff
@@ -166,9 +165,9 @@ fn transfer_fails_with_no_account() {
 		deploy(&mut session, "new", vec![TOKEN.to_string(), MIN_BALANCE.to_string()]).unwrap();
 	session.set_actor(contract.clone());
 	// `pallet-assets` returns `NoAccount` error.
-	assert_eq!(
+	assert_runtime_err!(
 		transfer(&mut session, ALICE, AMOUNT),
-		Err(into_psp22_custom(Module { index: 52, error: [1, 0] }))
+		DrinkError::Module(Assets(NoAccount))
```

Comment:
Not sure about the `DrinkError` name. I liked `RuntimeError` more now especially with the enum structure it has now.

Reply:
Resolved in https://github.com/r0gue-io/pop-node/pull/324/commits/ac65b7e5bc8e470c521bf5020a258a439979f872